planner/core/common_plans.go

```diff
@@ -743,6 +743,8 @@ func (e *Explain) explainPlanInRowFormat(p Plan, taskType, indent string, isLast
 			storeType = kv.TiKV.Name()
 		case kv.TiFlash:
 			storeType = kv.TiFlash.Name()
+		case kv.TiDB:
```

Comment:
I think we can refactor this part:
```go
	case *PhysicalTableReader:
		switch x.StoreType {
		case kv.TiKV, kv.TiFlash, kv.TiDB:
			// expected do nothing
		default:
			err = errors.Errorf("the store type %v is unknown", x.StoreType)
			return
		}
		storeType := x.StoreType.Name()
```

Reply:
And I have not seen you change the `Name` method implementation.

Reply:
Great, done.